app/imports/client/ui/dialogStack/DialogStack.vue

```diff
@@ -197,7 +197,7 @@
         // If the source and the hidden Element are different
         // hide the source and reveal the hidden element
         let originalSourceTransition = source.style.transition;
-        if (hiddenElement !== source){
+        if (hiddenElement && hiddenElement !== source){
```

Comment:
I added this because after a roll I got a `hiddenElement` undefined error which prevented the action buttons from resetting after the roll